scripts/docker/entrypoint.sh

```diff
@@ -6,7 +6,7 @@ cleanup() {
     echo "Terminating container processes..."
 
     # Terminate the main process if it has been started.
-    if [ -n "$MAIN_PID" ]; then
+    if [ -n "${MAIN_PID:-}" ]; then
```

Comment:
Since we `set -u`, this will fail if MAIN_PID is not defined (hitting it locally)